dataprofiler/tests/profilers/test_text_options.py

```diff
@@ -0,0 +1,26 @@
+from dataprofiler.profilers.profiler_options import TextOptions
+from dataprofiler.tests.profilers.test_numerical_options import TestNumericalOptions
+
+
+class TestTextOptions(TestNumericalOptions):
+    
+    option_class = TextOptions
+    keys = ["min", "max", "sum", "variance", "histogram_and_quantiles", "vocab"]
```

Comment:
i think you could do `keys = TestNumericalOptions.keys + ["vocab"]`
or do you think that becomes unclear?

Reply:
I think that's a good idea - if NumericalOptions keys change in the future, you would only need to edit one test file